data/han2020sequential/han2020sequential.yaml

```diff
@@ -0,0 +1,61 @@
+# yaml-language-server: $schema=../.schema.yaml
+title:
+  Sequential Analysis of Viral Load in a Neonate and Her Mother Infected With
+  Severe Acute Respiratory Syndrome Coronavirus 2
+description:
+  The study reports SARS-CoV-2 viral loads in different specimen types for a neonate and
+  her mother diagnosed on 2020-03-20. The study includes nasopharyngeal, oropharyngeal,
+  stool, plasma, saliva, and urine samples. Only concentrations in stool samples are
+  reported here. Viral loads were extracted manually from Figure 1 using
+  [WebPlotDigitizer](https://automeris.io).
+doi: 10.1093/cid/ciaa447
+analytes:
```

Comment:
This paper has nasopharynx, oropharynx, stool, saliva, plasma, and urine samples. Do we also want to include other sample types? One student can help with the extraction.

Reply:
Yes, I think that would be nice to include. Maybe we can do that in a follow-up PR?

Reply:
I've added an issue at #57.